packages/dtslint/src/index.ts

```diff
@@ -20,6 +20,8 @@ async function main(): Promise<void> {
   let lookingForTsLocal = false;
   let tsLocal: string | undefined;
 
+  console.log(`dtslint@${require("../package.json").version}`);
```

Comment:
is this helpful long-term?

Reply:
I think so. I added it when the version mismatch was just a hunch, not yet confirmed, and not yet mitigated with the CI check. But I think it’s useful to be able to look at older logs and tell what was running.